assets/js/workflow.js

```diff
@@ -195,8 +14,13 @@ class Workflow extends React.Component {
   constructor(props: iProps) {
     super(props);
     this.state = {
-      isPublic: false
+        isPublic: false,
+        focus: false,
+        overlapping: false,
```

Comment:
This would be a good place for a comment defining what "overlapping" means in all the code below